src/status_im/ui/screens/wallet/transactions/views.cljs

```diff
@@ -50,32 +56,36 @@
    [button/primary-button {:text (i18n/label :t/transactions-sign) :on-press #(on-sign-transaction m)}]
    [button/secondary-button {:text (i18n/label :t/delete) :on-press #(on-delete-transaction m)}]])
 
+(defn- inbound? [type] (= "inbound" type))
 (defn- unsigned? [type] (= "unsigned" type))
 
 (defn- transaction-icon [k background-color color]
   {:icon      k
    :icon-opts {:color color}
    :style     (transactions.styles/transaction-icon-background background-color)})
 
-(defn- transaction-type->icon [s]
-  (case s
-    "unsigned" (transaction-icon :icons/dots-horizontal styles/color-gray4-transparent styles/color-gray7)
-    "inbound" (transaction-icon :icons/arrow-left styles/color-green-3-light styles/color-green-3)
-    "outbound" (transaction-icon :icons/arrow-right styles/color-blue4-transparent styles/color-blue4)
-    ("postponed" "pending")  (transaction-icon :icons/arrow-right styles/color-gray4-transparent styles/color-gray7)
-    (throw (str "Unknown transaction type: " s))))
+(defn- transaction-type->icon [k]
+  (case k
+    :unsigned               (transaction-icon :icons/dots-horizontal styles/color-gray4-transparent styles/color-gray7)
+    :inbound                (transaction-icon :icons/arrow-left styles/color-green-3-light styles/color-green-3)
+    :outbound               (transaction-icon :icons/arrow-right styles/color-blue4-transparent styles/color-blue4)
+    (:postponed :pending)   (transaction-icon :icons/arrow-right styles/color-gray4-transparent styles/color-gray7)
+    (throw (str "Unknown transaction type: " k))))
 
 (defn render-transaction [{:keys [to from type value symbol] :as m}]
   [list/item
-   [list/item-icon (transaction-type->icon type)]
+   [list/item-icon (transaction-type->icon (keyword type))]
```

Comment:
Why this change?

Reply:
in order to reuse it for transaction-details as well, and keywords is more idiomatic